src/facades/author.js

```diff
@@ -1,15 +1,13 @@
 'use strict'
 
-let books = require('../../db.json')
+let { getBooks } = require('./book')
 
-function getAuthors () {
-  let result = new Set(
-    books
-      .map(book => book.authors)
-      .reduce((prev, curr) => prev.concat(curr))
-      .sort()
-    )
-  return [...result]
+function getAuthors() {
```

Comment:
It would be great to see an implementation of `getAuthors()` that involves our `lib/db` module since we want to utilize the LevelDB indexes. `lib/load_db` module already introduces few indexes that we can use:
- bookId => book
- category\x00{categoryName} => [ arrayOfBookIds ]
- categories => [ arrayOfDistinctCategories ]
- author\x00{authorName} => [ arrayOfBookIds ]
- authors => [ arrayOfDistinctAuthors ]